google/resource_bigtable_instance.go

```diff
@@ -44,11 +44,17 @@ func resourceBigtableInstance() *schema.Resource {
 			},
 
 			"num_nodes": {
-				Type:         schema.TypeInt,
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
```

Comment:
If we send a non-zero value to the server for `num_nodes` for a `DEVELOPMENT` instance, the request fails. There's no server default when you don't send a `num_nodes` for a production one; the request fails.

Users won't actually need to recreate their instances; the `num_nodes` of 3 is preserved in state, and they will just need to update their config when it diffs.